kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -608,6 +624,23 @@ private void verifyBucketSpecified() throws ConfigException {
     }
   }
 
+  private void checkAutoCreateTables() {
+    Class<?> schemaRetriever = getClass(BigQuerySinkConfig.SCHEMA_RETRIEVER_CONFIG);
+
+    boolean autoCreateTables = getBoolean(TABLE_CREATE_CONFIG);
+    if (autoCreateTables && schemaRetriever == null) {
+      throw new ConfigException(
```

Comment:
Nit: your exception has an extra four space indentation compared to rest of project. Same below, and same in line 116 of BigQuerySinkConnector.java

Reply:
Updated!